src/routes/volume/VolumeBulkActions.js

```diff
@@ -13,7 +13,8 @@ function bulkActions({ selectedRows, engineImages, bulkDeleteVolume, showBulkEng
           bulkDeleteVolume(selectedRows)
         } else {
           confirm({
-            title: `Are you sure you want to delete volume(s) ${selectedRows.map(item => item.name).join(', ')} ?`,
+            width: 700,
+            title: `The longhorn-manager will delete the Kubernetes resources associated with the Volume ${selectedRows.map(item => item.name).join(', ')},Are you sure you want to delete volume(s) ${selectedRows.map(item => item.name).join(', ')} ?`,
```

Comment:
Is there a way to check if any of the to be deleted `Volumes` are attached or have resources associated with them and to only append that to the error message if that's the case like for single `Volumes`?

If not, I think this should be rewritten to be:
```
Any resources (PersistentVolumes and PersistentVolumeClaims) associated with these volumes will be deleted. Any volumes currently attached to a node may cause errors in any applications using them once they are deleted! Are you sure you want to delete volume(s) ... ?
```